tests/conftest.py

```diff
@@ -49,6 +49,14 @@ def foreman_api(fixture_dir, requests_mock, tmpdir):
     return apypie.Api(uri='https://foreman.example.com', api_version=2, apidoc_cache_dir=tmpdir.strpath)
 
 
+@pytest.fixture
+def luna_api(fixture_dir, requests_mock, tmpdir):
+    with fixture_dir.join('luna.json').open() as read_file:
+        data = json.load(read_file)
+    requests_mock.get('https://foreman.example.com/apidoc/v2.json', json=data)
+    return apypie.Api(uri='https://foreman.example.com', api_version=2, apidoc_cache_dir=tmpdir.strpath)
```

Comment:
Wouldn't it be better to extract the example? Remote Execution might decide to change the interface...

Reply:
Well, we store the apidoc json statically in the fixtures, so it won't change unless we update it.
But I could minimize the apidoc?

Reply:
Oh right.
I read `requests_mock.get('https://foreman.example.com/apidoc/v2.json', json=data)` as
`requests.get('https://foreman.example.com/apidoc/v2.json')` and stopped thinking.